internal/cache/file/cache_handle_test.go

```diff
@@ -211,7 +213,7 @@ func (cht *cacheHandleTest) Test_IsSequential_WhenPrevOffsetGreaterThanCurrent()
 
 	isSeq := cht.cacheHandle.IsSequential(currentOffset)
 
-	ExpectEq(false, isSeq)
+	assert.Equal(cht.T(), false, isSeq)
```

Comment:
nit: assertFalse?
and other places also.

Reply:
done